lib/index.js

```diff
@@ -1,13 +1,26 @@
 var pkg = require('./pkg');
+var defaultInfo = require('./default-info');
 
 function info() {
-  var scripts = pkg()['scripts-info'];
+  var pkgJSON = pkg();
+  var scriptsInfo = pkgJSON['scripts-info'] || {};
 
-  if ( !scripts ) {
-    throw new Error('Oops, but no `scripts-info` property was found in your `package.json`');
+  Object.keys(pkgJSON.scripts || {})
```

Comment:
I think we should throw / warn if there are no scripts in the first place, since it's pointless to use this module without it haha